pkg/logs/config/config.go

```diff
@@ -149,13 +150,23 @@ func buildHTTPEndpoints() (*Endpoints, error) {
 		}
 		main.Host = host
 		main.Port = port
-		main.UseSSL = !coreConfig.Datadog.GetBool("logs_config.logs_no_ssl")
+		useSSL = !coreConfig.Datadog.GetBool("logs_config.logs_no_ssl")
 	default:
 		main.Host = coreConfig.GetMainEndpoint(httpEndpointPrefix, "logs_config.dd_url")
-		main.UseSSL = !coreConfig.Datadog.GetBool("logs_config.dev_mode_no_ssl")
+		useSSL = !coreConfig.Datadog.GetBool("logs_config.dev_mode_no_ssl")
+	}
+	main.UseSSL = useSSL
+
+	var additionals []Endpoint
+	err := coreConfig.Datadog.UnmarshalKey("logs_config.additional_endpoints", &additionals)
+	if err != nil {
+		log.Warnf("Could not parse additional_endpoints for logs: %v", err)
+	}
+	for i := 0; i < len(additionals); i++ {
+		additionals[i].UseSSL = useSSL
```

Comment:
np: WDYT of not introduce a local variable for `useSSL` and use explicitly the main configuration for the additional endpoints, i.e.: 
```suggestion
		additionals[i].UseSSL = main.useSSL
```

Reply:
👍